upload.php

```diff
@@ -180,7 +180,13 @@ public function set_filename($filename) {
 	 *
 	 * @return array
 	 */
-	public function upload($filename = '') {
+	public function upload($filename = false) {
```

Comment:
@sigsalerno can we change this to `null` instead of `false`? I think it's more appropriate. Thanks for the PR.